src/main/java/org/spongepowered/common/command/WrapperICommandSender.java

```diff
@@ -119,7 +120,7 @@ public MinecraftServer getServer() {
         if (world != null) {
             return world.getMinecraftServer();
         }
-        return null;
+        return SpongeImpl.getServer();
```

Comment:
`getEntityWorld` shouldn't ever return `null` so that was wrong in the first place